dj_gui_api_server/DJGUIAPIServer.py

```diff
@@ -219,6 +221,31 @@ def insert_tuple(jwt_payload: dict):
         return str(e), 500
 
 
+@app.route('/api/record/dependency', methods=['GET'])
+@protected_route
+def record_dependency(jwt_payload: dict) -> dict:
+    """
+    Route to insert record. Expects:
+        (html:GET:Authorization): Must include in format of: bearer <JWT-Token>
+        (html:query_params): {"schema_name": <schema_name>, "table_name": <table_name>,
+                           "restriction": <b64 JSON restriction>}
+            NOTE: Table name must be in CamalCase
+    :param jwt_payload: Dictionary containing databaseAddress, username and password
+        strings.
+    :type jwt_payload: dict
+    :return: If sucessfuly sends back a list of dependencies otherwise returns error
+    :rtype: dict
+    """
+    # Get dependencies
+    try:
+        dependencies = DJConnector.record_dependency(
+            jwt_payload, request.args.get('schema_name'), request.args.get('table_name'),
```

Comment:
```suggestion
            jwt_payload, request.args.get('schemaName'), request.args.get('tableName'),
```
same as above - to match query param format from the frontside